src/main/java/com/clickhouse/kafka/connect/sink/ClickHouseSinkConfig.java

```diff
@@ -23,6 +23,7 @@ public class ClickHouseSinkConfig {
     public static final String EXACTLY_ONCE = "exactlyOnce";
     public static final String SUPPRESS_TABLE_EXISTENCE_EXCEPTION = "suppressTableExistenceException";
     public static final String CLICKHOUSE_SETTINGS = "clickhouseSettings";
+    public static final String TABLE_MAPPING = "tableMapping";
```

Comment:
in bigquery connector it's called `topic2TableMap, in snowflake connector `topic2table.map`